### ISSUE TITLE
Display Error when unable to Connect

### ISSUE THREAD

**Issue by dns2utf8** [coredump-ch/ferraris-enumerator#2]
The system should inform the user about the failed connection. I propose:
- [ ] "No Net" => if WLan is not reacable or system does not have an ip
- [ ] "Offline" => if the server can not be found over DNS
- [ ] "No Service" => Server did not respond correctly

These Messages would then slide over the 4 7-Segment Displays

**Comment by dbrgn** [comment]
Alternatively, use Err1/Err2/Err3?

**Comment by dns2utf8** [comment]
Error Codes?

**Comment by dbrgn** [comment]
Yes. They fit on a screen, and the person that has to fix the problem would know what the problem is.

**Comment by dns2utf8** [comment]
We need a printed table hanging from the enumerator :)

**Comment by fmorgner** [comment]
My vote goes to simple error codes. Everything else is unnecessarily complicated.
